src/runtime_src/core/common/api/xrt_xclbin.cpp

```diff
@@ -604,6 +619,25 @@ class xclbin_full : public xclbin_impl
       : std::make_pair(nullptr, size_t(0));
   }
 
+  std::vector< std::pair<const char*, size_t> >
```

Comment:
Typically C++ style is std::vector<type> with no space for embedded type, e.g., just like you do below.